scripts/lib/api/processHtml.ts

```diff
@@ -432,20 +432,11 @@ function processAttribute(
 
 function processFunctionOrException(
```

Comment:
This also changes Exceptions. You should split those out into two separate helper functions.